internal/kafka/internal/api/public/model_supported_kafka_size.go

```diff
@@ -23,7 +25,14 @@ type SupportedKafkaSize struct {
 	// Maximum data retention period available to this Kafka instance size.
 	MaxDataRetentionPeriod string `json:"max_data_retention_period,omitempty"`
 	// Maximium connection attempts per second available to this Kafka instance size.
-	MaxConnectionAttemptsPerSec int32 `json:"max_connection_attempts_per_sec,omitempty"`
+	MaxConnectionAttemptsPerSec int32                            `json:"max_connection_attempts_per_sec,omitempty"`
+	MaxMessageSize              SupportedKafkaSizeBytesValueItem `json:"max_message_size,omitempty"`
+	// Minimum number of in-sync replicas.
+	MinInSyncReplicas int32 `json:"min_in_sync_replicas,omitempty"`
+	// Replication factor available to this Kafka instance size.
+	ReplicationFactor int32 `json:"replication_factor,omitempty"`
+	// List of Availability Zone modes that this Kafka instance size supports. The possible values are \"single\", \"multi\".
+	SupportedAzModes []string `json:"supported_az_modes,omitempty"`
```

Comment:
run `make code fix` so that the code gets formatted

Reply:
Hmmm... The field types should be aligned (see DisplayName, IngressThroughputPerSec, etc).
I believe there is a bug/limitation of the formatter when there is a comment between two fields.